packages/devextreme/js/__internal/ui/chat/chat.ts

```diff
@@ -53,14 +53,14 @@ class Chat extends Widget<Properties> {
 
     super._initMarkup();
 
+    this._renderMessageBox();
```

Comment:
I think that previous rendering order was correct.
If there is some issue with scrollable dimensions calculations, let's just update it after list rendering. But i don't like we try to hack rendering order. At least it will lead to difficulties when we start rendering header